web/src/winchan.d.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * This is not complete type definitions for the "Winchan" module, but it's enough
+ * for our purpose.
+ */
+declare module "winchan" {
```

Comment:
This should go in `web/src/types/winchan.d.ts` without the `declare module` line.